src/txdb.h

```diff
@@ -74,6 +74,8 @@ class CCoinsViewDB final : public CCoinsView
 class CCoinsViewDBCursor: public CCoinsViewCursor
 {
 public:
+    CCoinsViewDBCursor(CDBIterator* pcursorIn, const uint256 &hashBlockIn):
```

Comment:
Verified the ctor needs to be public to not see `error: calling a private constructor of class 'CCoinsViewDBCursor'` when building. Could clang-format while touching it:
```suggestion
    CCoinsViewDBCursor(CDBIterator* pcursorIn, const uint256& hashBlockIn):
```